### ISSUE TITLE
Is this really necessary? :)

### ISSUE THREAD

**Issue by soulman-is-good** [Amomentforever0/components#4]
https://github.com/Amomentforever0/components/blob/7a6f99eb0a7c7cbe72488fcbd3b2eae2ca6b7b6f/Tooltip/app.js#L29

**Comment by Amomentforever0** [comment]
No, looks like i need to put label in progress, and rethink the approach i tried to implement this feature :)
